conf/documentation.conf

```diff
@@ -1824,3 +1824,10 @@ options=enabled|disabled
 description=<<EOT
 Show the parking portal to the device instead of the usual portal
 EOT
+
+[liupf.deauthsourceip]
```

Comment:
Not sure where liupf comes from :)

It seems it would be better in `advanced`

Also, you need to add the parameter in pf.conf.defaults even if it will be empty